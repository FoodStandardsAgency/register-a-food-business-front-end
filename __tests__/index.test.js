import Index from "../pages/index.js";
import { shallow } from "enzyme";
import renderer from "react-test-renderer";
import * as emotion from "emotion";
import { createSerializer } from "jest-emotion";

expect.addSnapshotSerializer(createSerializer(emotion));

describe("<Index />", () => {
  it('shows "Hello world!"', () => {
    const wrapper = shallow(<Index />);
    expect(wrapper.find("p").text()).toEqual("Hello world!");
  });

  it("matches the previous snapshot", () => {
    const tree = renderer.create(<Index />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
