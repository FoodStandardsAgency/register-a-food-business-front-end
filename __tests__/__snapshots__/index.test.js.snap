// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Index /> matches the previous snapshot 1`] = `
<div>
  <div>
    <h1>
      Header
    </h1>
  </div>
  <p>
    Hello world!
  </p>
</div>
`;
